tests/CrEOF/Spatial/Tests/OrmTest.php

```diff
@@ -155,11 +160,15 @@ public static function setUpBeforeClass()
      */
     protected function setUp()
     {
+        if (count($this->supportedPlatforms) && ! in_array($this->getPlatform()->getName(), $this->supportedPlatforms, true)) {
+            $this->markTestSkipped(sprintf('No support for platform %s in test class %s.', $this->getPlatform()->getName(), get_class($this)));
+        }
+
         $this->entityManager = $this->getEntityManager();
         $this->schemaTool    = $this->getSchemaTool();
 
         if (true) {
-            static::getConnection()->executeQuery('SELECT 1 /*' . get_class($this) . '*/');
+            static::getConnection()->executeQuery(sprintf('SELECT 1 /*%s*//*%s*/', get_class($this), $this->getName()));
```

Comment:
Crap, I was going to make it configurable to turn off marking each test in
the DB query log. I should have added a todo. Suppose there's no reason not
to leave it on all the time. I'll fix and deal with the other commented
method too.

On Tue, Oct 27, 2015 at 1:27 PM, sadortun <notifications@github.com> wrote:

> In tests/CrEOF/Spatial/Tests/OrmTest.php
> <https://github.com/creof/doctrine2-spatial/pull/86#discussion_r43165140>:
>
> >          $this->entityManager = $this->getEntityManager();
> >          $this->schemaTool    = $this->getSchemaTool();
> >
> >          if (true) {
> > -            static::getConnection()->executeQuery('SELECT 1 /*' . get_class($this) . '*/');
> > +            static::getConnection()->executeQuery(sprintf('SELECT 1 /*%s*//*%s*/', get_class($this), $this->getName()));
>
> Are you testing that the connexion is alive ? You should probably add a
> comment, and/or remove the if(true)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/creof/doctrine2-spatial/pull/86/files#r43165140>.
>